src/services/peopleSearch.ts

```diff
@@ -75,6 +77,36 @@ export type PeopleSearchQuery = {
   involvement: string;
 };
 
+const getRenamedDepartments = async () => {
+  const dep = await getDepartments();
+  dep.forEach((d, i) => {
+    if (/^Office of the /.test(d)) {
+      dep[i] = dep[i].replace(/^Office of the /, '') + ' (Office of the)';
+    } else if (/^Office of /.test(d)) {
+      dep[i] = dep[i].replace(/^Office of /, '') + ' (Office of)';
+    } else if (/^Center for /.test(d)) {
+      dep[i] = dep[i].replace(/^Center for /, '') + ' (Center for)';
+    } else if (/^Department of /.test(d)) {
+      dep[i] = dep[i].replace(/^Department of /, '') + ' (Department of)';
+    }
+  });
+  console.log(dep);
+  return dep;
+};
+
+const getOldName = (dep: string) => {
+  if (dep.endsWith('(Office of the)')) {
+    dep = 'Office of the ' + dep.replace(/ \(Office of the\)/, '');
+  } else if (dep.endsWith('(Office of)')) {
+    dep = 'Office of ' + dep.replace(/ \(Office of\)/, '');
+  } else if (dep.endsWith('(Center for)')) {
+    dep = 'Center for ' + dep.replace(/ \(Center for\)/, '');
+  } else if (dep.endsWith('(Department of)')) {
+    dep = 'Department of ' + dep.replace(/ \(Department of\)/, '');
+  }
+  return dep;
+};
```

Comment:
is there a specific use case for this? I feel like we could just `getDepartments` again right? Feel free to correct me if I'm wrong. (or you could map `renamedDepartments` with the original `getDapartments` if you need both at the same time)

Reply:
```
const getRenamedDepartments = async () => {
  const dep = await getDepartments();
  const mappedResults = [];
  dep.forEach((d) => {
    let departmentName = { name: d, formattedName: d }
    if (/^Office of the /.test(d)) {
      departmentName.formattedName = d.replace(/^Office of the /, '') + ' (Office of the)';
    } else if (/^Office of /.test(d)) {
      departmentName.formattedName = d.replace(/^Office of /, '') + ' (Office of)';
    } else if (/^Center for /.test(d)) {
      departmentName.formattedName = d.replace(/^Center for /, '') + ' (Center for)';
    } else if (/^Department of /.test(d)) {
      departmentName.formattedName = d.replace(/^Department of /, '') + ' (Department of)';
    };
    mappedResults.push(departmentName);
  });
  return mappedResults;
};
```
you could probably replace the first function with something along the lines of this. I didn't write this function with linting or checking for syntax, so this could potentially be pseudo code.

Then outside of the service, when you pass the query back to the `search` route, you can just pass the original `name`